src/features/accuse/useAccusedWallets.ts

```diff
@@ -31,7 +31,7 @@ export function useAccusedWallets(
 
           // Only sign the messages if the values are valid
           if (!isBytes32(privateKey) || !isBytes32(sarcophagusId)) {
-            return;
+            continue;
```

Comment:
`return` was blocking the signatures list from resetting when the last valid key field is removed. This makes sure it happens, while not adding invalid sigs. Was worried about potential indexing flaws when using `continue` -- testing didn't reveal any obvious bugs I could identify, so left as is